op-challenger/game/scheduler/coordinator.go

```diff
@@ -64,8 +64,12 @@ func (c *coordinator) schedule(ctx context.Context, games []common.Address) erro
 		if j, err := c.createJob(addr); err != nil {
 			errs = append(errs, err)
 		} else if j != nil {
-			jobs = append(jobs, *j)
-			c.m.RecordGameUpdateScheduled()
+			if j.status == types.GameStatusInProgress {
```

Comment:
We should be doing this filtering in `createJob` and just not returning a job to schedule.

Reply:
Ah good catch!